CFC_DataCollector/recommender/improve_trip_pipeline.py

```diff
@@ -0,0 +1,28 @@
+######
+#1  Figure out best alternative for each trips
+#1.5 For each trip do:
+#3. Perturbe trips
+#4. Augment perturbed trips
+#5. Apply model to alternate and perturbed trips
+#6. Return list of improved trips
+
+#construction of model' can part of utility model pipeline or this pipeline
+
+#getAugmentedTrips: retrieve augmented trips from database, 
```

Comment:
High level comment: maybe you weren't quite clear on the notion of stubbing out functions, but basically what you want do is to create functions with that interface and simple/dummy implementations. So basically, everything should be importable and runnable without crashing. See Shaun's code (the first pipeline) for an example.